src/app/components/ContactView.scss

```diff
@@ -0,0 +1,24 @@
+@import "~design-system/_sass/reusable-components/design-system-config";
+
+.contactView-locked-background {
+    position: absolute;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    z-index: 2;
```

Comment:
are we sure this z-index is high enough? No possibility of other elements having a higher one?